business_rules/operators.py

```diff
@@ -257,9 +257,9 @@ class DataframeType(BaseType):
 
     name = "dataframe"
 
-    def __init__(self, value, column_prefix_map = {}):
-        self.value = self._assert_valid_value_and_cast(value)
-        self.column_prefix_map = column_prefix_map
+    def __init__(self, data):
```

Comment:
Does something in the business-rules expect a single argument here? Why can't have more than one arg?

Reply:
This line creates the actual operator: https://github.com/cdisc-org/business-rules/blob/4442811b2019ecbd125a168ddc1cc8ccbbad69fc/business_rules/engine.py#L75

if we want to provide more than one argument in some cases and only one in others, we’d need to do some type checking here and assume that the value is the first argument, I went this route to avoid type checking